clang/tools/c-index-test/c-index-test.c

```diff
@@ -8,7 +8,6 @@
 #include "clang-c/Documentation.h"
 #include "clang-c/Index.h"
 #include "clang/Config/config.h"
-#include "llvm/Support/AutoConvert.h"
```

Comment:
We need this header because we are using the function enablezOSAutoConversion below in this file

Reply:
This is a c++ header, if we really need it, then we need to update it to c. @abhina-sree Can you please help to do so? Thanks.

Reply:
Inside that header we guard the C++ code inside an `#ifdef __cplusplus` so it should work for both C and C++. Are you hitting any errors as a result of this header file? it is also guarded with `__MVS__` so no other platforms outside of z/OS should be using these functions

Reply:
Please update all the comments to use c style instead then?

Reply:
Yes, I think that would be a better solution, I will try to get to that when I have time, thanks!

Reply:
Done. Please review. Thanks.